ui/src/app/settings/components/project-details/project-details.tsx

```diff
@@ -42,90 +42,92 @@ function emptyMessage(title: string) {
+            merged.sourceRepos = merged.sourceRepos.concat(proj.spec.sourceRepos || []);
+            merged.destinations = merged.destinations.concat(proj.spec.destinations || []);
+
+            merged.sourceRepos = merged.sourceRepos.filter((item, index) => {
+                return (
+                    index ===
+                    merged.sourceRepos.findIndex(obj => {
+                        return obj === item;
+                    })
+                );
+            });
+
+            merged.destinations = merged.destinations.filter((item, index) => {
+                return (
+                    index ===
+                    merged.destinations.findIndex(obj => {
+                        return obj.server === item.server && obj.namespace === item.namespace;
+                    })
+                );
+            });
+
+            merged.clusterResourceBlacklist = merged.clusterResourceBlacklist.filter((item, index) => {
+                return (
+                    index ===
+                    merged.clusterResourceBlacklist.findIndex(obj => {
+                        return obj.kind === item.kind && obj.group === item.group;
+                    })
+                );
+            });
+
+            merged.clusterResourceWhitelist = merged.clusterResourceWhitelist.filter((item, index) => {
+                return (
+                    index ===
+                    merged.clusterResourceWhitelist.findIndex(obj => {
+                        return obj.kind === item.kind && obj.group === item.group;
+                    })
+                );
+            });
+
+            merged.namespaceResourceBlacklist = merged.namespaceResourceBlacklist.filter((item, index) => {
+                return (
+                    index ===
+                    merged.namespaceResourceBlacklist.findIndex(obj => {
+                        return obj.kind === item.kind && obj.group === item.group;
+                    })
+                );
+            });
+
+            merged.namespaceResourceWhitelist = merged.namespaceResourceWhitelist.filter((item, index) => {
+                return (
+                    index ===
+                    merged.namespaceResourceWhitelist.findIndex(obj => {
+                        return obj.kind === item.kind && obj.group === item.group;
+                    })
+                );
+            });
+            merged.count += 1;
+
+            return merged;
+        },
+        {
+            clusterResourceBlacklist: new Array<GroupKind>(),
+            namespaceResourceBlacklist: new Array<GroupKind>(),
+            namespaceResourceWhitelist: new Array<GroupKind>(),
+            clusterResourceWhitelist: new Array<GroupKind>(),
+            sourceRepos: [],
+            signatureKeys: [],
+            destinations: [],
+            description: '',
+            roles: [],
+            count: 0
+        }
     );
 }
 
+function loadGlobal(name: string) {
```

Comment:
Methods like `deleteJWTToken` still using `loadGlobal` but it is better to switch to `projects.getScoped` E.g.

```
await services.projects.deleteJWTToken(params);
const {project, globalProjects} = await services.projects.getScoped(this.props.match.params.name);
this.loader.setData([project, globalProjects]);
```

instead of 

```
await services.projects.deleteJWTToken(params);
const proj = await services.projects.get(this.props.match.params.name);
const globalProj = await loadGlobal(proj.metadata.name);
this.loader.setData([proj, globalProj]);
```

Can you please make this change and stop using `loadGlobal` in UI? So we can eventually deprecate and remove `loadGlobal` API